conformance/lang/expressions/let-expr/let_expr.balt

```diff
@@ -0,0 +1,481 @@
+Test-Case: output
```

Comment:
We seem to have combined multiple constructs in the same test. The usual pattern is to have separate tests as concise as possible. That way even an implementation that only supports a few constructs can run a set of tests.

Please check the already-merged tests.

Reply:
Changed